ldm/models/diffusion/plms.py

```diff
@@ -1017,7 +1028,7 @@ def plms_sampling_optimize_intrinsic_edit(
                     old_eps=old_eps,
                     t_next=ts_next,
                     input_image=input_image,
-                    optimizing_weight=torch.ones(50)[i],
+                    optimizing_weight=torch.ones(101)[i],
```

Comment:
Hi IlyaBeleckij,

Thank you very much for your help! I see that the previous code is limited to a fixed denoising step of 50, and this pull request tries to support arbitrary denoising steps. I am still a little bit wondering about line 1301 of plms.py (here), which changes the fixed value of 50 to 101. Does this mean the code would support a fixed denoising step of 101 in this case?

Reply:
Hi Wuqiuche, 
Thank you for your paper! It was awesome to get familiar with such concepts.
Yes, this is my fault from debugging. Honestly, it seems like we can just replace torch.ones(...)[i] with simple scalar one? Am I right?

Reply:
Hi IlyaBeleckij,

Yeah you are right, my previous code was tedious here.. I will merge your pull request and just change it to scalar one here. Thank you very much for your help again!